nexusparser/tools/dataconverter/readers/mpes/reader.py

```diff
@@ -307,18 +307,18 @@ def read(
 
                 # Filling in the metadata from xarray
                 elif precursor == "@attrs":
-
-                    try:  # Tries to fill the metadata
-                        template[key] = iterate_dictionary(
-                            x_array_loaded.attrs,
-                            value,
-                        )
-
-                    except KeyError:
-                        print(
-                            f"[info]: Path {key} not found. "
-                            f"Skipping the entry.",
-                        )
+                    if key not in eln_data_dict:
```

Comment:
I think we should refine that at one point. That we just fill in the keys present in the xarray and the keys in the eln and let the internal logic of the parser throw the errors if anything is missing.

But I think for now this hot fix is fine.